language/en.yml

```diff
@@ -247,23 +246,7 @@ miscs-help: |
   ×  /ping : Get bot latency.
   ×  /paste <flags> : paste text file to a pastebin service.
       flags:
-          [-d, -dogbin] = use dogbin (default)
-          [-n, -neko, -nekobin] = use nekobin
           [-h, -haste, -hastebin] = use hastebin\n
```

Comment:
There are a bit of changes on the `/paste` flags as from 09d0a31fdc80a43f50e6a095df73ecf16e7f4a4a the supported paste service are hastebin and katbin
with the flag `[-h, hastebin]` for hastebin and `[-k , katbin] for katbin`.

Reply:
Ok

Reply:
There are only 2 flags now [-h, hastebin] and [-k, katbin] (*the seccond flags don't have the (-) symbol)

Reply:
So, its only have 2 flags and katbin no need (-) symbol?

Reply:
Ok